### DISCUSSION TITLE
Adding more theme preference in Grafana

### DISCUSSION THREAD

**Discussion by a-imran99** [grafana/grafana#56489] [Alerting]
Hi @torkelo , I have been trying to add more theme preference alongside with Dark/Light themes, and probably did some modification and added them, But the problem is with the *isLight* Key from the API which is a boolean data type and can toggle between only 2 themes ( dark and light ).

So I have changed the data type to string and started to compare it on string level using the theme names ( isLight === 'hydra' ) . But this process seems to be quite complicated and I guess it should be done through the entire application.

Any thoughts or suggestions on this. it would be great if I get some ideas :) @torkelo 

Thanks

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
